kornia/augmentation/_2d/intensity/color_jitter.py

```diff
@@ -65,24 +73,31 @@ def __init__(
         same_on_batch: bool = False,
         p: float = 1.0,
         keepdim: bool = False,
-        return_transform: Optional[bool] = None,
+        return_transform: Optional[bool] = None
     ) -> None:
         super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, keepdim=keepdim)
+
+        warnings.warn(
```

Comment:
Can you also add an option to silent this warning? Might be annoying to some uers.

Reply:
Do you propose a flag at instantiation? I could add the flag `silence_instantiation_warning` if you want

Reply:
Yes. Anything like that may work.

Reply:
Ok it should be good now